docs/detections/prebuilt-rules/prebuilt-rules-downloadable-updates.asciidoc

```diff
@@ -11,6 +11,11 @@ To download the latest updates, follow the instructions in <<download-prebuilt-r
 |==============================================
 |Update version |Date | New rules | Updated rules | Notes
 
+|<<prebuilt-rule-0-16-2-prebuilt-rules-0-16-2-summary, 0.16.2>> | 28 Apr 2022 | 23 | 271 |
+This release includes new rules for MacOS, Linux and cloud services.
+Also, included are investigation guides for Windows rules and new rules for credential theft and Active Directory (AD).
+New threat-centric rules to help detect https://www.elastic.co/blog/detecting-and-responding-to-dirty-pipe-with-elastic[CVE-2022-0847 (Dirty Pipe)]
```

Comment:
Can you breakdown what was added to macos and gtfobins for linux (maybe not saying gtfobins 🙂 )

```suggestion
This release includes new rules for for MacOS, Linux and cloud services.
Also, included are expanded detailed investigation guides coverage for Windows rules as well as new rules for credential theft and Active Directory (AD).
Additionally, it includes new rules to help detect the emerging threat https://www.elastic.co/blog/detecting-and-responding-to-dirty-pipe-with-elastic[CVE-2022-0847 (Dirty Pipe)]
```